Tools/TestWebKitAPI/Tests/WebKitCocoa/ProcessSwapOnNavigation.mm

```diff
@@ -7848,6 +7848,20 @@ static void checkSettingsControlledByLockdownMode(WKWebView *webView, ShouldBeEn
     EXPECT_EQ(runJSCheck("!!window.PushSubscriptionOptions"_s), isShowingInitialEmptyDocument != IsShowingInitialEmptyDocument::Yes && shouldBeEnabled == ShouldBeEnabled::Yes); // Push API.
     String mathMLCheck = makeString("document.createElementNS('http://www.w3.org/1998/Math/MathML','mspace').__proto__ == ", shouldBeEnabled == ShouldBeEnabled::Yes ? "MathMLElement" : "Element", ".prototype");
     EXPECT_EQ(runJSCheck(mathMLCheck), true); // MathML.
+
+    // Confirm unstable Experimental settings are always off in Lockdown Mode.
+    // They are expected to always be off:
+    EXPECT_EQ(runJSCheck("CSS.supports('contain-intrinsic-size: 10rem')"_s), false);
```

Comment:
I imagine these checks would pass even without your change?

Shouldn't we enable all experimental features at the beginning of the test to actually make sure your code change is tested?

Something like:
```
    for (_WKExperimentalFeature *feature in [WKPreferences _experimentalFeatures])
            [[configuration preferences] _setEnabled:YES forExperimentalFeature:feature];
```